src/components/VoucherCard/index.tsx

```diff
@@ -46,7 +50,12 @@ const VoucherCard = ({
             )}`}</Text>
           </div>
 
-          <Text>Flash this eVoucher to redeem</Text>
+          {!isRedeemable && hasRedeemed &&
```

Comment:
Q: Is this intended? I'm a bit confused as to the purpose of `hasRedeemed` since this conditional handles non redeemable vouchers if I interpret it correctly

Reply:
I think i was also checking whether if the user has redeemed it, and if they have then change the "click" to "flash", aside from being assigned the unredeemable voucher (I've changed and to or)

Reply:
Ahh I see okay all good then!